content/id/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md

```diff
@@ -0,0 +1,284 @@
+# Sebagai superuser
+sudo sh -c "echo 'Hello from Kubernetes storage' > /mnt/data/index.html"
+```
+
+{{< note >}}
+Jika Node kamu menggunakan alat untuk akses _superuser_ selain dengan `sudo`, kamu dapat
+membuat ini bekerja jika mengganti `sudo` dengan nama dari alat lainya.
+{{< /note >}}
+
+Menguji bahwa berkas `index.html` ada:
+
+```shell
+cat /mnt/data/index.html
+```
+
+Keluaran akan seperti ini:
+```
+Hello from Kubernetes storage
+```
+
+Sekarang kamu dapat menutup _shell_ di Node kamu.
+
+## Membuat sebuah PersistentVolume
+
+Di latihan, kamu membuat sebuah *hostPath* PersistentVolume. Kubernetes mendukung
+hostPath untuk pengembangan dan pengujian di dalam klaster Node tunggal. Sebuah hostPath
+PersistentVolume menggunakan berkas atau direktori di dalam Node untuk meniru _network-attached storage_.
+
+Di dalam klaster _production_, kamu tidak dapat menggunakan hostPath. Sebagai gantinya sebuah administrator klaster
+akan menyediakan sumberdaya jaringan seperti Google Compute Engine _persistent disk_,
+_NFS share_, atau sebuah Amazon Elastic Block Store volume. Administrator klaster juga dapat 
+menggunakan [StorageClasses](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#storageclass-v1-storage)
+untuk mengatur
+[dynamic provisioning](https://kubernetes.io/blog/2016/10/dynamic-provisioning-and-storage-in-kubernetes).
+
+Berikut berkas konfigurasi untuk hostPath PersistentVolume:
+
+{{< codenew file="pods/storage/pv-volume.yaml" >}}
+
+Berkas konfigurasi tersebut menentukan bahwa volume berada di `/mnt/data` pada 
+klaster Node. Konfigurasi tersebut juga menentukan ukuran dari 10 gibibytes dan 
+mode akses `ReadWriteOnce`, yang berarti volume dapat di pasang sebagai
+_read-write_ oleh Node tunggal. Konfigurasi ini menggunakan [StorageClass name](/docs/concepts/storage/persistent-volumes/#class)
+`manual` untuk PersistentVolume, yang akan digunakan untuk mengikat
+permintaan PeristentVolumeClaim ke PersistentVolume ini.
+
+Membuat sebuah PersistentVolume:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/storage/pv-volume.yaml
+```
+
+Melihat informasi tentang PersistentVolume:
+
+```shell
+kubectl get pv task-pv-volume
+```
+
+Keluaran menunjuk PersistentVolume memliki sebuah `STATUS` dari `Available`. Ini
+berarti PersistentVolume belum terikat ke PersistentVolumeClqaim.
+
+    NAME             CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS      CLAIM     STORAGECLASS   REASON    AGE
+    task-pv-volume   10Gi       RWO           Retain          Available             manual                   4s
+
+## Membuat sebuah PersistentVolumeClaim
+
+Langkah selanjutnya adalah membuat sebuah PersistentVolumeClaim. Pods menggunakan PersistentVolumeClaims
+untuk meminta penyimpanan fisik. Dalam latihan ini, kamu akan membuat sebuah PersistentVolumeClaim
+yang meminta sebuah volume setidaknya tiga gibibytes dengan mode akses _read-write_ 
```

Comment:
"setidaknya" is repeated, can use other word
```suggestion
yang meminta sebuah volume minimal tiga gibibytes dengan mode akses _read-write_ 
```

Reply:
btw mas is it ok to keep gibibytes not in italic?

Reply:
I think it's fine, since it's a metric unit